assets/scss/components/stencil/heroCarousel/_heroCarousel.scss

```diff
@@ -28,18 +24,15 @@
         margin-top: -(spacing("single") + spacing("base")); // 3
     }
 
-    .js & { // 2
```

Comment:
I thought at first that we don't need this, but it does serve a purpose on IE -- let's leave it in.

Reply:
Oh? What are you seeing?

We definitely have to at least modify it, left as-is it causes pop-in as the element slowly transitions to its full size.